client/src/components/container/admin/controlPage/MainPage/index.js

```diff
@@ -0,0 +1,69 @@
+import React ,{Component} from "react";
+import {Button} from "antd"
+import Axios from 'axios'
+import { PlusSquareFilled } from '@ant-design/icons';
+
+import "./style.css"
+
+class Dashboard extends Component {
+    state={
+        name:"",
+        img:"",
+        error:"",
+        renderView:"add"
+    }
+    componentDidMount(){
+        Axios.get('/api/v1/admin').then(({data})=>{
+            const {name,img}=data[0]
+            this.setState({name,img})
+        }).catch(err=>{
+            this.setState({error:err})
+        })
+    }
+    logout=()=>{
+        Axios.get('/api/v1/logout').then((res)=>{
+            this.props.history.push('/login');
+        }).catch(err=>{
+            this.setState({error:err.msg})
+        })
+    }
+
+    clickBtn = e => {
+        this.setState({renderView: e.target.value});
+    }
```

Comment:
use destructing here : 
```suggestion
    clickBtn = ({target:{value}}) => {
        this.setState({renderView: value});
    }
```